src/Debug/Hoed.hs

```diff
@@ -93,6 +95,8 @@ import           Language.Haskell.TH
 import           Language.Haskell.TH.Syntax
 import           System.Clock
 
+{-# ANN module ("hlint: ignore Redundant bracket" :: String) #-}
```

Comment:
What triggers this warning? Could you not just remove the redundant bracket instead?

Reply:
Conditional compilation:
```
      ([ ViewPatterns
       , PartialTypeSignatures
       , ExtendedDefaultRules
       , FlexibleContexts
       ]
#if __GLASGOW_HASKELL__ >= 802
       ++
       [DeriveAnyClass | generateObservableInstances] ++
       [DerivingStrategies | generateObservableInstances] ++
       [DeriveGeneric | generateGenericInstances]
#endif
      )
``

Reply:
Yuk, but fair enough.